python_modules/dagster/dagster/_core/utils.py

```diff
@@ -196,3 +197,12 @@ def submit(self, fn, *args, **kwargs):
 def is_valid_email(email: str) -> bool:
     regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b"
     return bool(re.fullmatch(regex, email))
+
+
+class StrictModel(BaseModel):
```

Comment:
leave a lil doc block with some explanation of why standardizing on these settings